xdcs-server/src/main/java/pl/edu/agh/xdcs/services/TaskService.java

```diff
@@ -162,13 +177,40 @@ public TaskCreationWizard addResourcePattern(
             return this;
         }
 
+        public TaskCreationWizard addKernelArguments(List<File> kernelArguments) {
+            this.kernelArguments = new HashMap<>();
+            for (int i = 0; i < kernelArguments.size(); i++) {
+                try {
+                    byte[] bytes = FileUtils.readFileToByteArray(kernelArguments.get(i));
+                    String objectId = objectRepository.store(Blob.fromBytes(bytes));
```

Comment:
jak czytasz z pliku to możesz użyć `BlobStream`, zaoszczędzisz ładowania do pamięci całości

Reply:
nieaktualne